doc/source/release-notes/Percona-Server-5.5.59-38.11.rst

```diff
@@ -0,0 +1,34 @@
+.. rn:: 5.5.59-38.11
+
+=============================
+|Percona Server| 5.5.59-38.11
+=============================
+
+Percona is glad to announce the release of |Percona Server| 5.5.59-38.11 on
+January 30th, 2018. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.59-38.11/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+
+Based on `MySQL 5.5.59
+<http://dev.mysql.com/doc/relnotes/mysql/5.5/en/news-5-5-59.html>`_, including
+all the bug fixes in it, |Percona Server| 5.5.59-38.11 is now the current
+stable release in the 5.5 series. All of |Percona|'s software is open-source
+and free.
+
+Bugs Fixed
+==========
+
+ With :variable:`innodb_large_prefix` set to ``1``, Blackhole storage engine
+ was incompatible with InnoDB table definitions, thus adding new indexes would
+ cause replication erros on the slave. Fixed :psbug:`1126` (upstream
+ :mysqlbug:`53588`).
+
+ GCC 7 fixes introduced regression in |Percona Server| :rn:`5.5.58-38.10` when
```

Comment:
A GCC 7 warning fix introduced ... , also would be good to describe the circumstances, when Federated was used how